zero-module.yml

```diff
@@ -70,6 +70,12 @@ parameters:
     options:
       - "postgres"
       - "mysql"
+  - field: cacheStore
```

Comment:
Let's make sure this matches the definition in aws-eks-stack

Reply:
will use three options for all the places: "redis" / "memcached" / "".